diesel/src/sqlite/connection/mod.rs

```diff
@@ -436,6 +472,36 @@ mod tests {
     use crate::prelude::*;
     use crate::sql_types::Integer;
 
+    #[test]
+    fn database_deserializes_successfully() {
+        let connection = &mut SqliteConnection::establish(":memory:").unwrap();
+        let data = include_bytes!("sqlite_test.db");
```

Comment:
It would be nice to not include an actual database in the git repository. We could just add support for `sqlite3_serialize` and do a roundtrip here instead (so first serialize then deserialize).